Creating labels seems to require new permission

#### Describe the bug

Running your action causes the following error:

```
Run # Label PR.
gh: You do not have permission to create labels on this repository. (HTTP 403)
Error: Process completed with exit code 1.
```

According to https://github.com/googleapis/release-please-action/issues/1105#issuecomment-2780292262 adding `issues: write` solves this.

```diff
permissions:
  actions: read        # Required to identify workflow run.
  checks: write        # Required to add status summary.
  contents: read       # Required to checkout repository.
  pull-requests: write # Required to add comment .
+ issues: write        # Required to add label.
```

#### To Reproduce

1. Create a workflow with these permissions and this step:
```yaml
jobs:
  deploy:
    permissions:
      actions: read
      checks: write
      contents: read
      pull-requests: write

    steps:
     ...
      - uses: op5dev/tf-via-pr@v13
        with:
          working-directory: terraform
          command: ${{ github.event_name == 'pull_request' && 'plan' || 'apply' }}
          validate: true
          arg-lock: ${{ github.event_name != 'pull_request' }}
```
2. Run workflow
3. Error happens

#### Expected behavior

Job shouldn't fail when creating the label.

Documentation and examples mention additional required permission.

--- a/action.yml
+++ b/action.yml
@@ -159,6 +159,7 @@ runs:
         ${{ inputs.tool }}${{ steps.arg.outputs.arg-chdir }} validate${args} 2> >(tee tf.console.txt) > >(tee tf.console.txt)
 
     - if: ${{ inputs.label-pr == 'true' && steps.identifier.outputs.pr != 0 && contains(fromJSON('["plan", "apply"]'), inputs.command) }}
+      continue-on-error: true
       shell: bash
       run: |
         # Label PR.

